app/src/main/java/com/cooper/wheellog/utils/InMotionAdapter.java

```diff
@@ -1302,7 +1299,7 @@ public static String toHexString(byte[] buffer) {
                             outValues.add(infos);
                         }
                     } else if (result.id == CANMessage.IDValue.PinCode.getValue()) {
-                        passwordSent = true;
+                        passwordSent = 100;
```

Comment:
magic

Reply:
если просто большое число надо, то лучше как-то так
```suggestion
                        passwordSent = Integer.MAX_VALUE;
```